src/ReplyBuilder.js

```diff
@@ -75,22 +75,22 @@ export const buildRecipients = (envelope, ownAddress) => {
 	let to = []
 	let cc = []
 	if (recipientType === RecipientType.To) {
-		// Send to everyone except yourself plus the original sender
+		// Send to everyone except yourself, plus the original sender if not ourself
 		to = envelope.to.filter(isNotOwnAddress)
-		to = to.concat(envelope.from)
+		to = to.concat(envelope.from.filter(isNotOwnAddress))
```

Comment:
oh that makes sense! Could we add a test case for this? :v:

Reply:
done :)